media-gfx/viewnior/viewnior-1.7.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit xdg-utils gnome2-utils meson
+
+MY_PN=Viewnior
+DESCRIPTION="Fast and simple image viewer"
+HOMEPAGE="https://siyanpanayotov.com/project/viewnior/ https://github.com/hellosiyan/Viewnior"
+SRC_URI="https://github.com/hellosiyan/${MY_PN}/archive/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-libs/glib:2
```

Comment:
I'm not sure what you mean by this. `gio` is part of glib, `x11-libs/gdk-pixbuf` is a dependency of `x11-libs/gtk+:2`. Should I add `virtual/pkgconfig` to `DEPEND`?

Reply:
>Should I add virtual/pkgconfig to DEPEND?

Yes. I also wanted you to confirm that the other deps gets pulled, since I didn't know ;)

For what it matters, looks good to me. Builds and works fine.

Reply:
I have updated the ebuild.